.github/workflows/shorebird_ci.yml

```
@@ -0,0 +1,41 @@
+name: shorebird_ci
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+
+    runs-on: ${{ matrix.os }}
+
+    name: 🐦 Test
```

```suggestion
    name: 🐦 Shorebird Test
```
nit just for clarity